packages/workspace/src/browser/workspace-service.ts

```diff
@@ -120,11 +122,11 @@ export class WorkspaceService implements FrontendApplicationContribution {
     }
 
     protected reloadWindow(): void {
-        window.location.reload();
+        this.windowHelper.reloadWindow(window);
```

Comment:
is it necessary? i found 2 other places in code, there `window.location.reload()` is called, both in `@theia/core`. To make it right, we need to move `WindowHelper` in core.